src/main/java/org/folio/tlib/postgres/impl/PgCqlQueryImpl.java

```diff
@@ -0,0 +1,270 @@
+package org.folio.tlib.postgres.impl;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.tlib.postgres.PgCqlField;
+import org.folio.tlib.postgres.PgCqlQuery;
+import org.z3950.zing.cql.CQLBooleanNode;
+import org.z3950.zing.cql.CQLNode;
+import org.z3950.zing.cql.CQLParseException;
+import org.z3950.zing.cql.CQLParser;
+import org.z3950.zing.cql.CQLPrefixNode;
+import org.z3950.zing.cql.CQLSortNode;
+import org.z3950.zing.cql.CQLTermNode;
+
+public class PgCqlQueryImpl implements PgCqlQuery {
+  private static final Logger log = LogManager.getLogger(PgCqlQueryImpl.class);
+
+  final CQLParser parser = new CQLParser(CQLParser.V1POINT2);
+  final Map<String, PgCqlField> fields = new HashMap<>();
+
+  String language = "english";
```

Comment:
It means it's the only stemming that's supported right now. IIRC RMB has a setting for this, but AFAIK has never been set to anything else.

Reply:
OK, sounds fine to me. Maybe worth a comment.

Reply:
RMB hard-codes `simple` because English stemming gives bad results for other languages like Spanish, French, German, ...:
https://github.com/folio-org/raml-module-builder/blob/v33.0.2/domain-models-runtime/src/main/resources/templates/db_scripts/general_functions.ftl#L131-L136

Libraries usually have items in different languages and some FOLIO libraries are non-English. I haven't heard of any complain about `simple`.